packages/create/templates/hello-typeorm/README.md

```diff
@@ -0,0 +1,49 @@
+# DBOS Hello with TypeORM
+
+This is a [DBOS app](https://docs.dbos.dev/) bootstrapped with `npx @dbos-inc/dbos-sdk init` and using [TypeORM](https://docs.dbos.dev/tutorials/using-typeorm).
+
+## Getting Started
+
+Before you can launch your app, you need a database.
+DBOS works with any Postgres database, but to make things easier, we've provided a script that starts a Docker Postgres container and creates a database.
+Run:
+
+```bash
+node start_postgres_docker.js
+```
+
+If successful, the script should print `Database started successfully!`.
+
+Next, build the app:
+
+```bash
+npm run build
+```
+
+Then, run a schema migration to create some tables:
+
+```bash
+npx dbos-sdk migrate
+```
+
+If successful, the migration should print `Migration successful!`.
```

Comment:
While it is possible to figure out how building incremental migration works from the docs, I think a paragraph about it, and example within the DBOS context, would help quite a bit.  Even a sentence about how it was done would help.

Reply:
I can add some light documentation of migrations to this readme with a link to more